packages/shared/forks/ReactFeatureFlags.www-dynamic.js

```diff
@@ -27,6 +27,7 @@ export const enableLazyContextPropagation = __VARIANT__;
 export const enableSyncDefaultUpdates = __VARIANT__;
 export const consoleManagedByDevToolsDuringStrictMode = __VARIANT__;
 export const warnOnSubscriptionInsideStartTransition = __VARIANT__;
+export const enableSuspenseAvoidThisFallback = __VARIANT__;
```

Comment:
I think we can probably hard code this in given that Fizz is not on anyway, there's nothing to test. And I'm not sure it make much sense to test how something not working performs.

Reply:
Gotcha, I wasn't sure exactly how __VARIANT__ works but sounds like it automatically makes a GK?

Reply:
It doesn't automatically do it but www-dynamic is designed for us to manually wire up a GK. If you hard code it in ReactFeatureFlags.www.js instead, it'll be compiled out from runtime. The goal is for everything to eventually move out of www-dynamic to www once we remove the GKs.